it/uk/gov/hmrc/transitmovementsrouter/connectors/PushNotificationConnectorSpec.scala

```diff
@@ -130,5 +135,16 @@ class PushNotificationConnectorSpec
       }
     }
 
+    "when the service is disabled, return a unit" in {
+      when(mockAppConfig.pushNotificationsEnabled).thenReturn(false)
+
+      stub(OK)
```

Comment:
In thie scenario, the stub should never be hit -- can we change this to `INTERNAL_SERVICE_ERROR` so that if it gets hit, we'll get a Left and know the test failed?